### PR TITLE
feat: adds statistics icon

### PR DESCRIPTION
**Pull request by dapeduu**
Adds the statistics card

## Did you use styled-components?

- [x] Yes
- [ ] No

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## How to test

Import and use it on any Profile page

```
import StatisticsCard from "components/moleculars/StatisticsCard";
<StatisticsCard
        icon="TicketIconOutlined"
        description="Tickets adoados"
        totalDonated="50"
        isMember
      />
```

### CODE REVIEW

File: src/components/moleculars/StatisticsCard/styles.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    }
+  }};
+  border-radius: 16px;
+  padding: 24px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  gap: 50px;
+  width: 80%;
+`;
```
Comment (thaisribon) [inline src/components/moleculars/StatisticsCard/styles.ts:22]: I think the gap and width: 80% are not necessary, instead you can use max-width: 280px

File: src/components/moleculars/StatisticsCard/styles.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+export const Left = styled.View``;
```
Comment (thaisribon) [inline src/components/moleculars/StatisticsCard/styles.ts:24]: in the Left you should add:
```
  display: flex;
  alignItems: start;
  justifyContent: center;
```

File: src/components/moleculars/StatisticsCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import RibonFlagIcon from "components/vectors/RibonFlagIcon";
+import * as S from "./styles";
+
+type IconType = "TicketIconOutlined" | "RibonFlagIcon";
+
+type Props = {
+  totalDonated: string;
+  description: string;
+  icon: IconType;
+  isMember: boolean;
```
Comment (thaisribon) [inline src/components/moleculars/StatisticsCard/index.tsx:11]: it is good to add background color on props because we will need that

Comment (dapeduu) [reply src/components/moleculars/StatisticsCard/index.tsx]: Done!

File: src/components/moleculars/StatisticsCard/styles.ts (hunk 2 above)
Comment (dapeduu) [reply src/components/moleculars/StatisticsCard/styles.ts]: Done!

File: src/components/moleculars/StatisticsCard/styles.ts (hunk 1 above)
Comment (dapeduu) [reply src/components/moleculars/StatisticsCard/styles.ts:21]: Done!

**Review by thaisribon — APPROVED** [review]

**Pull request merged**